examples/evaluation/huggingface-evaluation.ipynb

```diff
@@ -0,0 +1,1340 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "b479f115-ac85-4701-8fa7-cade63376b74",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "# Evaluate a HuggingFace LLM with mlflow.evaluate()\n",
+    "\n",
+    "This guide will show how to load a pre-trained HuggingFace pipeline, log it to MLflow, and use `mlflow.evaluate()` to evaluate builtin metrics as well as custom LLM-judged metrics for the model.\n",
+    "\n",
+    "For detailed information, please read the following documentation:\n",
+    "https://mlflow.org/docs/latest/llms/llm-evaluate/index.html\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Start MLflow Server"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can either:\n",
+    "\n",
+    "- Start a local tracking server by running `mlflow ui` within the same directory that your notebook is in\n",
+    "  - Please follow [this section of the contributing guide](https://github.com/mlflow/mlflow/blob/master/CONTRIBUTING.md#javascript-and-ui) to get the UI set up.\n",
```

Comment:
We don't need to send people here. This is more for setting up for UI development.